content/docs/command-reference/exp/run.md

```diff
@@ -84,13 +84,23 @@ steps as `make_checkpoint()` — please refer to its reference for details.
 You can now use `dvc exp run` to begin the experiment. If the process gets
 interrupted (e.g. with `[Ctrl] C` or by an error), all the checkpoints so far
 will be preserved. When a run finishes normally, a final checkpoint will be
-added (if needed) to wrap up the experiment.
+added (if needed) to wrap up the experiment. Following uses of `dvc exp run`
+will continue from this point (using the latest cached versions of all outputs).
```

Comment:
Had to re-read this several times to understand the original wording. How about these synonyms which can't be misinterpreted?

```suggestion
added (if needed) to wrap up the experiment. Subsequent uses of `dvc exp run`
will resume from this point (using the latest cached versions of all outputs).
```